update-locale.sh

```diff
@@ -9,11 +9,9 @@ xgettext -j ./*.js -o "$pot" --from-code UTF-8 --no-wrap
 xgettext -j preferences/*.js -o "$pot" --from-code UTF-8 --no-wrap
 xgettext -j schemas/*.xml -o "$pot" --from-code UTF-8 --no-wrap
 
-for locale_lang in locale/*; do
-    po="$locale_lang/LC_MESSAGES/gnome-shell-extension-caffeine.po"
+for po in locale/*; do
     echo "$po"
     msgmerge --backup=off -U "$po" "$pot"
-    msgfmt "$po" -o "${po%po}mo"
```

Comment:
Shouldn't we remove mo files from locale dir as well ?

I see the po files are `locale/lang.po` instead of `locale/lang/LC_MESSAGES/gnome-shell-extension-caffeine.po`, that's something required by gnome-extensions pack?

Reply:
> I see the po files are locale/lang.po instead of locale/lang/LC_MESSAGES/gnome-shell-extension-caffeine.po, that's something required by gnome-extensions pack?

As far as I know, yes. That's what I've done for my other extensions, and that's been working. I'm assuming the old structure was required because the extension was being zipped manually, but since `gnome-extensions pack` handles `.mo` creation, it picks its own structure.

> Shouldn't we remove mo files from locale dir as well ?

I'm not sure what you mean, I can't see any `.mo` files in `locale/`?

Reply:
Sorry there no `mo` files, they are ignored by git